actionmailbox/lib/action_mailbox/errors.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+require "active_support/actionable_error"
+require "rails/command"
```

Comment:
Action Mailbox doesn't depend on Railties, so we can't just straight up require this here.